src/SupportBee/SupportBee.php

```diff
@@ -178,4 +179,20 @@ public function webhooks()
 	{
 		return WebHooks::webhooks();
 	}
+
+	public function avg_first_response_time_report( $options = array() )
```

Comment:
@levi730  Can you rename the methods to camel case as `avgFirstResponseTimeReport`? Its the pattern followed for methods.